Python/rerf/RerF.py

```diff
@@ -52,6 +55,11 @@ def fastRerF(
     fractionOfFeaturesToTest : float, optional
         Sets mtry based on a fraction of the features instead of an 
         exact number (default: None).
+    binMin : int, optional
+        If set to greater than 0, this is used as the minimum number of
+        samples in the node for which to use subsampled points.
+    binSize : int, to be used with binMin
```

Comment:
is binSize the number of bins or the number of samples in each bin?

Reply:
number of points in each bin. Confirmed from James.